rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.86"
+channel = "nightly"
```

Comment:
why?

Reply:
`sui-move-build` needs this when bumped to `mainnet-v1.61.2` - it's only needed in tests however, I just wasn't sure how to separate the 2 verisons